codegen/model/resource.go

```diff
@@ -60,6 +61,19 @@ type Resource struct {
 	Kind   string
 	Spec   Field
 	Status *Field
+	// external package where types can be found, leave empty if same as group
+	Package *ExternalPackage
+	// The set of additional printer columns to apply to the CustomResourceDefinition
+	AdditionalPrinterColumns []apiextv1beta1.CustomResourceColumnDefinition
+}
+
+type ExternalPackage struct {
+	// The go package where the status and spec field can be found, if different than group root
+	// leave empty if same as output dir
+	GoPackage string
+	// The relative path to the package where the spec and status field can be found
+	// leave empty if same as output dir
+	RelativePath string
```

Comment:
let's discuss options for removing this

Reply:
Oh yea, I forgot about that, I will fix later